Packs/IntegrationsAndIncidentsHealthCheck/.pack-ignore

```diff
@@ -1,2 +1,8 @@
 [file:playbook-JOB_-_Integrations_and_Playbooks_Health_Check.yml]
 ignore=PB105
+[file:JOB - Integrations and Playbooks Health Check]
```

Comment:
you have to provide file name (file path)
like in the first line

Reply:
Done